src/service/wallet.ts

```diff
@@ -84,6 +89,13 @@ export const makeWalletService = () => {
 
     if (isWalletConnectionInProgress || chainConnection) return;
 
+    if (
+      latestTermsIndex !== localStorageStore.getState().latestTermsAgreedIndex
```

Comment:
consider having something in the dialog (maybe in small gray print) that the terms must be agreed to in each browser. to stave off support requests about having to agree repeatedly. 

do we check anywhere that localStorage is available? we might need to error when it's disabled. people may auto-clear it but not much we can do about that

Reply:
I'm not sure exactly how to word that, especially considering multiple ipfs hash domains, but I can follow up and see if that's more helpful or just more confusing.

Looks like if localstorage isn't available, the app just crashes because `atomWithStorage` throws. I'm reluctant to try and handle that because it's pretty out of the ordinary, and at least for chrome, you'll see a "site not working?" tip whenever it's off by default.